setup.py

```diff
@@ -112,7 +112,8 @@ def run_tests(self):
         'requests>=2.6',
         
         'qrcode',
-        'twisted'
+        'twisted',
+        'six',
```

Comment:
hrm.
This is probably not entirely correct as babelglade is needed during setup phase rather than install phase.
It's only needed for replacing the check for `unicode`. I'm wondering how much this check is needed anyway.
`self.source` is the fileobj given by babel.  Babel seems to open that file in binary mode so it doesn't return `unicode` (python2) nor `str` (python3).
I'm hopeful that the three lines or so with the unicode check can be removed and with it the issue of getting another dependency in the right place.
Can you investigate whether removing that check causes problems?

Reply:
You are right, the type will always be "str" for Python 2 and "bytes" for Python 3. So I removed the check and the six dependency.